postcss.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     require('postcss-inline-svg')({
       paths: ['./src/assets'],
     }),
+    require('postcss-color-hex-alpha')(),
```

Comment:
Is this still necessary now that `colormin` is disabled?

Reply:
This makes it so that if we author CSS using 4 or 8 digit hex codes, it'll get transpiled to rgba(). I don't know for sure whether we are or aren't using that, but I think there is not much harm in having this plugin there in case we do in the future.

Reply:
I'd rather drop the dependency, as I don't see us using hex alpha colors manually very much.

Reply:
✅